### DISCUSSION TITLE
Volume control

### DISCUSSION THREAD

**Discussion by Fideluser94** [MonitorControl/MonitorControl#1688] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

Hi, so my question is in regards to the volume control. I have the app installed on a M1 Mac Pro, the Mac is connected to an OLED LG TV C3 2023.

The brightness can be controlled from the keyboard and the menu bar, works perfectly. 

The volume shows the slider and moves up and down, but the actual volume of the TV is not changing.. that is the issue, could you please help me with fixing this? 

I'm using a Logitech keyboard currently, but my Apple Magic Keyboard is on the way, thought I would mention it, maybe that is the issue.. 

I also thought it could be a TV setting that needs to be turned on..?

My Mac and TV are connected through h a USB-C to HDMI cable. 

To summarize, everything works and shows EXCEPT the volume of the TV actually changing.

**Comment by waydabber** [comment]
Most TV-s do not support DDC so volume control won't work. For brightness probably only the 0-50% range works, you need to disable DDC under Settings for the TV to make it work properly.

**Comment by time-less-ness** [comment]
This problem occurs on a Samsung 4k TV I'm using. I also seem only able to control 50% of the brightness as mentioned by @waydabber. But the max brightness I can get to seems bright enough for me, so I don't mind that.

FWIW, my TV seems to have no DDC or such setting anywhere I can find. The `Sound` menu is pretty sparse, and other menus seem to have nothing appropriate.

I don't expect any fixes here, just wanted to report that Samsung TVs have the problem(s), too.
